scripts/release/deploy_staging_jars.sh

```diff
@@ -36,16 +36,37 @@ if [ "$#" -gt "1" ]; then
   exit 1
 fi
 
+BUNDLE_MODULES=$(find -s packaging -name 'hudi-*-bundle' -type d)
+BUNDLE_MODULES_EXCLUDED="-${BUNDLE_MODULES//$'\n'/,-}"
+
 declare -a ALL_VERSION_OPTS=(
-"-Dscala-2.11 -Dspark2 -Dflink1.13" # for legacy bundle name
-"-Dscala-2.12 -Dspark2 -Dflink1.13" # for legacy bundle name
-"-Dscala-2.12 -Dspark3 -Dflink1.14" # for legacy bundle name
-"-Dscala-2.11 -Dspark2.4 -Dflink1.13"
-"-Dscala-2.11 -Dspark2.4 -Dflink1.14"
-"-Dscala-2.12 -Dspark2.4 -Dflink1.13"
-"-Dscala-2.12 -Dspark3.3 -Dflink1.15"
-"-Dscala-2.12 -Dspark3.2 -Dflink1.14"
-"-Dscala-2.12 -Dspark3.1 -Dflink1.14" # run this last to make sure utilities bundle has spark 3.1
+# upload all module jars and bundle jars
+"-Dscala-2.11 -Dspark2.4 -pl $BUNDLE_MODULES_EXCLUDED"
+"-Dscala-2.12 -Dspark2.4 -pl $BUNDLE_MODULES_EXCLUDED"
+"-Dscala-2.12 -Dspark3.3 -pl $BUNDLE_MODULES_EXCLUDED"
+"-Dscala-2.12 -Dspark3.2 -pl $BUNDLE_MODULES_EXCLUDED"
+"-Dscala-2.12 -Dspark3.1"  # this profile goes last in this section to ensure bundles use avro 1.8
```

Comment:
So, this command is where we build all other bundles is it? (kafka, presto, trino etc) 
is there diff versions required for any of these like we have for utilities bundle?

Reply:
yes. they all use avro 1.8.2 (same as spark 3.1 profile sets). this is also to retain the same build profiles as previously set (see the old last command)